index.html

```diff
@@ -0,0 +1,156 @@
+            <span class="icon">
+              <img src="./assets/images/forum.png" alt="forum" />
+            </span>
+            <h4 class="title">Forum</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/workshop.png" alt="workshop" />
+            </span>
+            <h4 class="title">Workshop</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/ignite.png" alt="ignite" />
+            </span>
+            <h4 class="title">Irreechaa Ignite</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+        </div>
+        <button class="join-btn">Join Erriecha Festival 2023</button>
+        <div class="see-program">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+    </main>
+    <section class="featured-speaker" id="dynamicPage">
+      
+    </section>
+
+    <footer class="footer" id="foo">
+      <section class="footer-section">
+        <h3 class="footer-title">Partner</h3>
+        <ul>
+          <li class="contributer-images">
+            <img class="mozilla" src="./assets/images/partner/mozilla.png" alt="mozilla" />
+          </li>
+          <li class="contributer-images">
+            <img class="google" src="./assets/images/partner/google.png" alt="google" />
+          </li>
+          <li class="contributer-images">
+            <img class="naver" src="./assets/images/partner/naver.png" alt="naver" />
+          </li>
+          <li class="contributer-images">
+            <img class="cisco" src="./assets/images/partner/airbnb.png" alt="" />
+          </li>
+          <li class="contributer-images">
+            <img class="airbnb" src="./assets/images/partner/microverse.png" alt="microverse" />
+          </li>
+        </ul>
+      </section>
+      <div class="logo-and-description">
+        <div class="logo"><a href="index.html"><img src="./assets/images/titleicon.png" class="linklogo-image"></a></div>
+        <div class="description">
+          <p>Irreechaa  Love,Peace and Unity</p>
+          <p id="desktop">
+            Irreechaa is usually celebrated during the weekend following the Meskel holiday. The festival is observed in many areas of Oromia, but, at the national level, it brings millions of Oromos from all over Oromia and non-Oromo visitors from other parts of Ethiopia to the shores of Hora (Lake) Arsadi and, as of 2019, to Hora Finfinee in the capital. 
+          </p>
```

Comment:
- [ ] As suggested by previous reviewer, you're required to stick to the guiding design as close as possible. This includes color, background-color, layout, padding, margin among others. You must persionalize the content, but the layout, color, layout must remain. Kindly walk your design alongside the guiding design.
  -  currently the partner companies font sizes are smaller so try fixing it.
  -  the bottom footer background color is different from Behance design so try fixing it.
your design
![ty](https://user-images.githubusercontent.com/96683843/226132641-551dec1a-791e-46ab-9123-d2d00ca67c23.jpg)